src/codegen/pass2_generate.cc

```diff
@@ -177,6 +177,27 @@ class GenEnumElem : public RenderCallback {
     FORBID_COPY(GenEnumElem);
 };
 
+class GenYytargetElem : public RenderCallback {
+    std::ostream& os;
+    const std::string& base;
+    const std::string& label;
+
+  public:
+    GenYytargetElem(
+            std::ostream& os, const std::string& base, const std::string& label)
+        : os(os), base(base), label(label) {}
+
+    void render_var(StxVarId var) override {
+        switch (var) {
+        case StxVarId::BASE: os << base; break;
+        case StxVarId::LABEL: os << label; break;
```

Comment:
Formatting: in a simple switch where each case fits on one line we do indent cases.